v21.2/take-and-restore-locality-aware-backups.md

```diff
@@ -24,7 +24,7 @@ A locality-aware backup is specified by a list of URIs, each of which has a `COC
 The locality query string parameters must be [URL-encoded](https://en.wikipedia.org/wiki/Percent-encoding).
 {{site.data.alerts.end}}
 
-Since each node is responsible for backing up the ranges for which it is the [leaseholder](architecture/replication-layer.html#leases), the locality of the leaseholder node determines where the backup files will be placed in a locality-aware backup. The leaseholder node writes files to the backup storage location with a locality that matches its own localities (with an overall preference for more specific values in the locality hierarchy). If there is no match, the `default` locality is used.
+Every node involved in the backup is responsible for backing up the ranges for which it was the [leaseholder](architecture/replication-layer.html#leases) at the time the distributed backup flow was planned. The locality of the node running the distributed backup flow determines where the backup files will be placed in a locality-aware backup. The coordinating node running the flow, and the leaseholder node of the range being backed up are usually the same, but can differ when lease transfers have occurred during the execution of the backup. The leaseholder node returns the files to the node running the backup flow (usually a local transfer), which then writes the file to the external storage location with a locality that matches its own localities (with an overall preference for more specific values in the locality hierarchy). If there is no match, the `default` locality is used.
```

Comment:
nit: instead of using the term coordinating which could be confused with the coordinator of the job, lets just stick to `The node running the flow, and the leaseholder node...`